xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/DeleteSpaceAction.java

```diff
@@ -33,7 +33,7 @@
     @Override
     protected boolean delete(XWikiContext context) throws XWikiException
     {
-        return deleteToRecycleBin(context.getDoc().getDocumentReference().getLastSpaceReference(), context);
+        return deleteToRecycleBin(context.getDoc().getDocumentReference().getLastSpaceReference(), context, true);
```

Comment:
Why true there? It's supposed to always skip the recycle bin?

Reply:
Exact, fixed here https://github.com/xwiki/xwiki-platform/pull/1369/files#diff-89f46c0930cfd7a08734b237f37a14ecR36